### ISSUE TITLE
Set Logger to AFLoggerLevelDebug

### ISSUE THREAD

**Issue by pjebs** [AFNetworking/AFNetworkActivityLogger#36]
For the v3.0.0 branch, is the documentation correct to change the logger level to debug?

XCode seems to say that setLogger doesn't exist. The docs say it does.
yet the docs also state at the top some other way to set the logger.

**Comment by ahmozkya** [comment]
I'm not sure if it's true, but I'm using like this (for 3_0_0 branch)

~~~objc
#ifdef DEBUG
    AFNetworkActivityConsoleLogger *consoleLogger = [AFNetworkActivityConsoleLogger new];
    [consoleLogger setLevel:AFLoggerLevelDebug];
    [[AFNetworkActivityLogger sharedLogger] removeLogger:[[[AFNetworkActivityLogger sharedLogger] loggers] anyObject]];
    [[AFNetworkActivityLogger sharedLogger] addLogger:consoleLogger];
    [[AFNetworkActivityLogger sharedLogger] startLogging];
#endif
~~~

**Comment by aaronschachter** [comment]
I've gotten `AFLoggerLevelDebug` working with similar code, setting the level of the `AFNetworkActivityConsoleLogger` in the sharedLogger's `loggers` property per the [README](https://github.com/AFNetworking/AFNetworkActivityLogger/tree/3_0_0#logging-levels)

````
AFNetworkActivityConsoleLogger *logger = [AFNetworkActivityLogger sharedLogger].loggers.anyObject;
logger.level = AFLoggerLevelDebug;
[[AFNetworkActivityLogger sharedLogger] startLogging];
```
Confirmed by debugging in `AFNetworkActivityConsoleLogger` that the `logger.level` is set to 1 [`AFLoggerLevelDebug`](https://github.com/AFNetworking/AFNetworkActivityLogger/blob/4faefc20f1a6489e88e2f2ee72b18d4727b3ebd8/AFNetworkActivityLogger/AFNetworkActivityLogger.h#L27), and the headers are logged.

My error logging isn't returning as much as it used to on 2.0, but investigating whether its a new `AFNetworking` 3.0 change in the way a `NSError.userInfo` is returned.

**Comment by SlaunchaMan** [comment]
This should be fixed by #43 in our 3.0.0 release!
